src/braket/default_simulator/state_vector_simulation.py

```diff
@@ -131,16 +137,28 @@ def density_matrix(self) -> np.ndarray:
     @property
     def state_with_observables(self) -> np.ndarray:
         """
-        np.ndarray: The state vector diagonalized in the basis of the measured observables.
+        State vector with observables applied.
 
         Raises:
             RuntimeError: If observables have not been applied
+
+        Returns:
+            np.ndarray: The state vector diagonalized in the basis of the measured observables.
```

Comment:
IIRC, properties are actually documented the previous way

Reply:
Good call, reverting